best-practices/README.md

```diff
@@ -104,11 +104,18 @@ Rails
 Testing
 -------
 
+(See also the rspec style guide that appears in this section)
+
+* First #describe what you are doing.
+* Then establish the #context.
+* #it only expects one thing (within reason).
```

Comment:
Yes indeed - the _it_ should describe one thing but can contain multiple expectations. Will fix. 

Also, @JamesFerguson thanks for reminding me - I can remember Brad teaching me that bit :)